src/ui/Header.tsx

```diff
@@ -28,7 +30,7 @@ export default function Header(props: HeaderProps) {
       left={0}
       right={0}
       height={`${HEADER_HEIGHT}px`}
-      zIndex="sticky"
+      zIndex="popover"
```

Comment:
I'm not sure this should be done, since the header really is "sticky" and not a popover. I think if there are z-index issues I'd rather fix them some other way

Reply:
My intention was to make sure the TOC popover is always placed at the top(currently resides under the Header Component) instead of the PageButton component. Should I update the page button instead?

Reply:
Yea I think that's probably the right move. We could also make the TOC popover a higher z index and the header one can stay sticky I think. But yes, for the styles PR